files/tomcat_init

```diff
@@ -5,8 +5,22 @@
 # chkconfig: - 90 10
 # description: Tomcat is a Java application Server.
 
+### BEGIN INIT INFO
+# Provides: a Tomcat instance
+# Required-Start: $network $syslog
+# Required-Stop: $network $syslog
+# Default-Start:
+# Default-Stop:
+# Short-Description: Start up the Tomcat server daemon
+# Description:       Tomcat is a Java application Server.
+#                    This service starts up the Tomcat server daemon.
+### END INIT INFO
+
 NAME=`/bin/basename $0`
 
+# if this is a symlink from rc#.d, the S## part is not useful, filter it out
+NAME=`/bin/sed 's/^S[0-9]*//g' <<< $NAME`
```

Comment:
after adding the LSB-style header to the init script, on my CentOS 6-based test machine, it refused to start Tomcat, throwing the following error in boot.log: Could not source a configuration file for the S90dspace Tomcat application. The "S90dspace" was the clue I needed, the NAME=`bin/basename $0` code was returning the wrong name, when run in CentOS6. I debated a few approaches to dealing with this problem, one of which was to completely overwrite the init script. However, I opted for this fix instead. If it's objectionable, I'll go with overwriting the initi script.